frontend/src/App.tsx

```diff
@@ -56,17 +56,19 @@ import PasswordResetSuccessPage from "./components/auth/PasswordResetSuccess";
 import NewAccountPage from "./components/pages/NewAccountPage";
 import AccountCreatedPage from "./components/pages/AccountCreatedPage";
 
-ReactGA.initialize(process.env.TRACKING_ID ?? "");
+// Consts for Hotjar and Google Analytics (this is ok to expose)
+const TRACKING_ID = "G-DF2BP4T8YQ";
+const HJID = "2949419";
+const HSJV = "6";
```

Comment:
```suggestion
const HJID = 2949419;
const HSJV = 6;
```

If we're just using them as ints, let's just instantiate them as such